lib/script/script.go

```diff
@@ -2,15 +2,37 @@ package script
 
 import (
 	"fmt"
+	"log"
 	"os"
 	"os/exec"
 	"strings"
 
 	"github.com/Originate/git-town/lib/git"
+	"github.com/Originate/git-town/lib/util"
 
 	"github.com/fatih/color"
 )
 
+var browserTools = []string{"xdg-open", "open"}
+var missingBrowserToolMessages = []string{
+	"Opening a browser requires 'open' on Mac or 'xdg-open' on Linux.",
+	"If you would like another command to be supported,",
+	"please open an issue at https://github.com/Originate/git-town/issues",
+}
+
+func OpenBrowser(url string) {
+	for _, browserTool := range browserTools {
+		if util.GetCommandOutput("which", browserTool) != "" {
```

Comment:
What about extracting a `getOpenTool` function that returns the name  of  the "open" tool for the current platform?